Homework8_2/RealEstateObjectExceptions.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace Homework8_2
+{
+    public class RealEstateObjectExceptions : Exception
+    {
+        private string? _address;
```

Comment:
These fields are never used. So you just pass it to base constructor. You can create different consts of error messages to pass to base constructor as well

Reply:
yep, removed